iphone/Maps/UI/PlacePage/PlacePageLayout/Content/BookingCells/PPHotelDescriptionCell.swift

```diff
@@ -8,19 +8,29 @@ final class PPHotelDescriptionCell: MWMTableViewCell
   @IBOutlet private weak var buttonZeroHeight: NSLayoutConstraint!
   @IBOutlet private weak var button: UIButton!
   private weak var updateDelegate: MWMPlacePageCellUpdateProtocol?
+  private var isNeedToForceLayout: Bool = false
 
   func config(with description: String, delegate: MWMPlacePageCellUpdateProtocol) {
     descriptionText.text = description;
     descriptionText.sizeToFit()
     updateDelegate = delegate
 
-    let isCompact = descriptionText.height > kMaximumDescriptionHeight;
-    if (isCompact) {
-      compactModeConstraints.forEach { $0.priority = UILayoutPriorityDefaultHigh }
-    }
-
-    hideButton(!isCompact)
     setNeedsLayout()
+    isNeedToForceLayout = true
+    layoutIfNeeded()
```

Comment:
func layoutIfNeeded()
Description
Lays out the subviews immediately.
Use this method to force the layout of subviews before drawing

func setNeedsLayout()
Description	
Invalidates the current layout of the receiver and triggers a layout update during the next update cycle.